src/windows/models/emoji_model.py

```diff
@@ -153,68 +158,45 @@ def update_model(self, clear=True):
 
                 row = []
 
-                # Append thumbnail
-                col = QStandardItem()
-                col.setIcon(QIcon(thumb_path))
-                col.setText("%s (%s)" % (emoji_name, emoji_type))
-                col.setToolTip(emoji_name)
-                col.setData(emoji_type)
-                col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
-                row.append(col)
-
-                # Append Filename
+                # Set emoji data
                 col = QStandardItem("Name")
-                col.setData(emoji_name, Qt.DisplayRole)
+                col.setIcon(QIcon(thumb_path))
                 col.setText(emoji_name)
+                col.setToolTip(emoji_name)
+                col.setData(path)
                 col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
                 row.append(col)
 
-                # Append Media Type
-                col = QStandardItem("Type")
-                col.setData(type, Qt.DisplayRole)
-                col.setText(emoji_type)
-                col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
-                row.append(col)
-
-                # Append Path
-                col = QStandardItem("Path")
-                col.setData(path, Qt.DisplayRole)
-                col.setText(path)
-                col.setFlags(Qt.ItemIsSelectable | Qt.ItemIsEnabled | Qt.ItemIsUserCheckable | Qt.ItemIsDragEnabled)
+                # Append filterable group
+                col = QStandardItem(emoji_type)
                 row.append(col)
 
                 # Append ROW to MODEL (if does not already exist in model)
-                if not path in self.model_paths:
+                if path not in self.model_paths:
                     self.model.appendRow(row)
                     self.model_paths[path] = path
 
-        # Get default emoji filter group
-        s = get_settings()
-        default_type = s.get('emoji_group_filter') or 'smileys-emotion'
-
-        # Loop through emoji groups, and populate emoji filter drop-down
-        get_app().window.emojiFilterGroup.clear()
-        get_app().window.emojiFilterGroup.addItem(_("Show All"), "Show All")
-        dropdown_index = 1
-        for emoji_type in sorted(emoji_groups.keys()):
-            get_app().window.emojiFilterGroup.addItem(_(emoji_type.capitalize()), emoji_type)
-            if emoji_type == default_type:
-                # Initialize emoji filter group to settings
-                get_app().window.emojiFilterGroup.setCurrentIndex(dropdown_index)
-            dropdown_index += 1
-
```

Comment:
The dropdown list really shouldn't be populated from the model — that's a view function. I moved this to the view, and just moved to storing a list of strings in a class member `self.emoji_groups` that it references.

(By doing it in the view, and waiting until after the signals are connected, another advantage is that we don't have to worry about manually refreshing — the `setCurrentIndex()` on the dropdown box automatically triggers `refresh_view()`.)